projects/pastanaga/src/lib/badge/badge.component.ts

```diff
@@ -20,7 +20,7 @@ import { Avatar } from '../avatar/avatar.model';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class BadgeComponent implements AfterViewInit {
-
+    @Input() id = '';
```

Comment:
well, we are doing it quite a lot in our projects actually: inputs like `required`, `disabled`... I feel like when we are using an input to fill a corresponding attribute, we should keep the real attribute name.

Reply:
It's not a big deal just a recommendation, just keep in mind that using that input we are passing the value through Angular input and setting the native property of the wrapper element. This can lead to some errors, let me explain with a little example:
`<pa-badge id="random-id"></pa-badge>`
Then in the component template we use that input to set the id of a native `span`:
`<span [id]="id"></span>`
If we render that component and use `document.getElementById('random-id')` we will get the `pa-badge` wrapper instead of the `span` element.

Reply:
Actually you are wrong, angular is smart enough. In your example, `document.getElementById('random-id')` will return the `span` and not the `pa-badge` (I just checked 😉)

Reply:
I think you have checked using Angular reflection: `[id]="'random-id'"`, use the native way instead: `id="random-id"`. I just checked in `ui-demo` and it's behaving as I mentioned :thinking: